src/extensions/core/load3d.ts

```diff
@@ -18,7 +18,35 @@ import { generateUUID } from '@/utils/formatUtil'
 
 useExtensionService().registerExtension({
   name: 'Comfy.Load3D',
-
+  settings: [
+    {
```

Comment:
Suggested change to make it obvious that the setting is only for initial value.

```
{
  id: 'Comfy.Load3D.ShowGrid',
  category: ['Load 3D', 'Scene', 'Show Grid'],
  name: 'Initial Grid Visibility',
  tooltip: 'Controls whether the grid is visible by default when a new 3D widget is created. This default can still be toggled individually for each widget after creation.',
  type: 'boolean',
  defaultValue: true,
  experimental: true
},
```

Reply:
done

Reply:
Please change setting names to
- Initial Grid Visibility
- Initial Preview Visibility
- Initial Camera Type

Reply:
done